### PR TITLE
Update Pagerduty landing page and widget images

### PR DESCRIPTION
**Pull request by pkwok**
### What does this PR do?

Updates the landing page image and widget image of the PagerDuty integration

### Motivation

We updated the UI of the integration so wanted the images to reflect the new UI.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by hithwen** [comment]
Pinging @light-Keeper as integration maintainer

**Review by sarah-witt — APPROVED** [review]
Looks good for agent integrations

File: pagerduty/manifest.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "configuration": "README.md#Setup",
     "support": "README.md#Support",
     "changelog": "CHANGELOG.md",
-    "description": "Monitor Pagerduty services and interact with incidents directly from Datadog dashboard",
+    "description": "Monitor your PagerDuty incidents from your Datadog dashboard",
```
Comment (pkwok) [inline pagerduty/manifest.json:21]: Had to make this under 70 characters as it was failing a test due to length being too long. I think Datadog updated the description length limit to be shorter than it was before.

**Review by sarah-witt — APPROVED** [review]

**Comment by anawishnoff** [comment]
Hi PagerDuty team, thanks for opening this PR! Since your tile contains a UI extension, it needs to follow our new guidelines for media carousel image requirements - specifically, images need to be 1440px in width and have a 16:9 aspect ratio. You can read more about the [guidelines here](https://docs.datadoghq.com/developers/marketplace/#media-carousel-requirements), let us know if you have any questions.

**Comment by pkwok** [comment]
> Hi PagerDuty team, thanks for opening this PR! Since your tile contains a UI extension, it needs to follow our new guidelines for media carousel image requirements - specifically, images need to be 1440px in width and have a 16:9 aspect ratio. You can read more about the [guidelines here](https://docs.datadoghq.com/developers/marketplace/#media-carousel-requirements), let us know if you have any questions.

Hi @anawishnoff I've updated the images to follow the guidelines :)

**Review by anawishnoff — APPROVED** [review]
Thanks @pkwok - LGTM!

**Comment by anawishnoff** [comment]
Hi @pkwok just one more thing - I noticed there was no release notes/changelog entry for this update. If there's new functionality added, you may want to add a release note that would appear on the tile - you can do this by editing the `changelog.md` file. 

It's up to you and your team on if you want to add a release note for these changes (maybe they are too minor), but wanted to confirm before we publish. Thanks!

**Comment by pkwok** [comment]
Thanks for reminding me of the changelog @anawishnoff We've added that now :)

File: pagerduty/CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # CHANGELOG - PagerDuty
 
 1.0.0 / 2021-10-20
+1.1.0 / 2022-02-03
 ==================
 ### Changes
 
 * [FEATURE] Status Dashboard by PagerDuty widget
 * [FEATURE] Incidents by PagerDuty widget
```
Comment (anawishnoff) [inline pagerduty/CHANGELOG.md:9]:
```suggestion
```

File: pagerduty/CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,11 @@
 # CHANGELOG - PagerDuty
 
 1.0.0 / 2021-10-20
+1.1.0 / 2022-02-03
```
Comment (anawishnoff) [inline pagerduty/CHANGELOG.md:4]:
```suggestion
## 1.1.0 / 2022-02-03
```

File: pagerduty/CHANGELOG.md [hunk 4]
```diff
@@ -1,8 +1,11 @@
...
 ==================
 ### Changes
 
 * [FEATURE] Status Dashboard by PagerDuty widget
 * [FEATURE] Incidents by PagerDuty widget
+* [FEATURE] Updated UI for the Incidents by PagerDuty widget. Widget now also displays Escalation Policies.
```
Comment (anawishnoff) [inline pagerduty/CHANGELOG.md:10]:
```suggestion
 * [FEATURE] Updated UI for the Incidents by PagerDuty widget. Widget now also displays Escalation Policies.
```

**Comment by anawishnoff** [comment]
@pkwok Sorry for the delay - I've added some suggestions to get the changelog file properly organized

**Review by anawishnoff — APPROVED** [review]

**Pull request merged**